File: duplicate-assert/java-junit/CheckstyleAntTaskTest_ testCreateClasspath.java
import static org.junit.jupiter.api.Assertions.assertEquals;

import org.junit.jupiter.api.Test;

public class CheckstyleAntTaskTest extends AbstractPathTestSupport {

    @Test
    public void testCreateClasspath() {
        final CheckstyleAntTask antTask = new CheckstyleAntTask();

        assertEquals("", antTask.createClasspath().toString(), "Invalid classpath");

        antTask.setClasspath(new Path(new Project(), "/path"));

        assertEquals("", antTask.createClasspath().toString(), "Invalid classpath");
    }
}

File: duplicate-assert/python-unittest/CheckstyleAntTaskTest_ testCreateClasspath.py
import unittest

class CheckstyleAntTaskTest(unittest.TestCase):

    def test_create_classpath(self):
        ant_task = CheckstyleAntTask()

        self.assertEqual("", ant_task.create_classpath().__str__(), "Invalid classpath")

        ant_task.set_classpath(Path(Project(), "/path"))

        self.assertEqual("", ant_task.create_classpath().__str__(), "Invalid classpath")
